evanderkoogh/otel-cf-workers
# specify output dir in dist

updated tsconfig for esm + cjs to explicitly set output dirs; added package info embed to avoid package.json path resolution issue

Fixes #162 

**What this PR solves / how to test:**

- specify output dirs in tsconfig for esm + cjs
- use build-time generated js file for name + version instead of package.json resolution

testing env:

- node v20.16.0
- typescript@5.4.4

to test:

1. create a cloudflare worker via `npm create cloudflare@latest -- my-first-worker`
2. `cd my-first-worker`
3. confirm worker runs with `npm run dev`
4. install latest otel cf workers with `npm i @microlabs/otel-cf-workers`
5. run `npm run dev`, should see error per #162
